installer/darwin/customize_dmg.applescript

```diff
@@ -0,0 +1,73 @@
+-- This script is derived from the create-dmg utility
+-- https://github.com/create-dmg/create-dmg
```

Comment:
Copied from this file and hard coded everything : https://github.com/create-dmg/create-dmg/blob/master/support/template.applescript